### DISCUSSION TITLE
Communication problem between N.I.N.A and Stellarium

### DISCUSSION THREAD

**Discussion by krakus240856** [Stellarium/stellarium#3472] [Q&A]
Hi, I use Stellarium from N.I.N.A. While working, I receive a message that NINA cannot connect to Stellarium, make sure it is running (and it is) and configured to receive requests.
The second problem is that when working in NINA I want to go to Stelarium, then it disconnects.

**Comment by gzotti** [comment]
Although I have never used NINA out of lack of time, "configured to receive requests" certainly means "activate and switch on the RemoteControl plugin" as is also hopefully described in NINA's manuals. 

If "disconnects" actually means "Stellarium quits", this is a known issue found months ago. It still awaits solution by somebody familiar with NINA and Stellarium, able to compile Stellarium and able to run a debugger. I had hoped for a somewhat higher percentage within our EAA users to become actively involved. 

The current Qt5-based releases of Stellarium seem to be less affected by issues related to telescope control, you may try that one for now.

**Comment by alex-w** [comment]
The version 23.3 should not be affected by this issue.

**Reply by gzotti** [reply]
Ah right, SharpCap users reported crashes.
